code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -108,7 +108,7 @@
 			var/obj/item/clothing/glasses/G = I
 			if(G.glass_colour_type)
 				update_glasses_color(G, 1)
-			if(G.tint)
+			if(G.tint || initial(G.tint))
```

Comment:
why, exactly, do we care about initial(G.tint) and in the other places that check for initial(G.tint)?

Reply:
like, if the tint of the mask has been removed, what is the update_tint() proc gonna do?